CHANGELOG.md

```diff
@@ -30,6 +30,7 @@ All notable, unreleased changes to this project will be documented in this file.
 - Drop `Django.Auth` - #11305 by @fowczarek
 - Propagate voucher discount between checkout lines when charge_taxes is disabled - #11632 by @maarcingebala
 - Accept the gift card code provided in the input - by @mociepka
+- Fix trigger the `FULFILLMENT_APPPROVED` webhook for partial fulfillments - #11816 by @d-wysocki
```

Comment:
This will have to be moved to 3.12 section once [this](https://github.com/saleor/saleor/pull/11818/) is merged.

Reply:
I removed it since it's a port to 3.10, I added changes to changelog in this [PR](https://github.com/saleor/saleor/pull/11824)